specs/eventing/data-plane.md

```diff
@@ -1,125 +1,179 @@
-and
-[version 0.3](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md)
-with restrictions and extensions specified below.
+- Knative Eventing aims to make writing
+  [event sources](./overview.md#event-source) and event-processing software
+  easier to write; as such, it imposes higher standards on system components
+  like [brokers](./overview.md#broker) and [channels](./overview.md#channel)
+  than on edge components.
 
-### HTTP Support
+This contract defines a mechanism for a single event sender to reliably deliver
+a single event to a single recipient. Building from this primitive, chains of
+reliable event delivery and event-driven applications can be built.
 
-This section adds restrictions on
-[requirements in HTTP Protocol Binding for CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/http-protocol-binding.md#12-relation-to-http).
+## Background
 
-Sinks MUST accept HTTP requests with POST method and MAY support other HTTP
-methods. If a method is not supported Sink MUST respond with HTTP status code
-`405 Method Not Supported`. Non-event requests (e.g. health checks) are not
-constrained.
-
-The URL used by a Sink MUST correspond to a single, unique endpoint at any given
-moment in time. This MAY be done via the host, path, query string, or any
-combination of these. This mapping is handled exclusively by the
-[Addressable control-plane](./interfaces.md#control-plane) exposed via the
-`status.address.url`.
-
-If an HTTP request's URL does not correspond to an existing endpoint, then the
-Sink MUST respond with `404 Not Found`.
-
-Every non-Callable Sink MUST respond with `202 Accepted` if the request is
-accepted.
-
-If Sink is Callable it MAY respond with `200 OK` and a single event in the HTTP
-response. A returned event is not required to be related to the received event.
-The Callable should return a successful response if the event was processed
-successfully. If there is no event to send back then Callable Sink MUST respond
-with 2xx HTTP and with empty body.
-
-If a Sink receives a request and is unable to parse a valid CloudEvent, then it
-MUST respond with `400 Bad Request`.
-
-### Content Modes Supported
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in RFC2119.
 
-A Sink MUST support `Binary Content Mode` and `Structured Content Mode` as
-described in
-[HTTP Message Mapping section of HTTP Protocol Binding for CloudEvents](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#3-http-message-mapping)
+When not specified in this document, the
+[CloudEvents HTTP bindings, version 1.0](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md)
+and [HTTP 1.1 protocol](https://tools.ietf.org/html/rfc7230) standards should be
+followed (with the CloudEvents bindings preferred in the case of conflict).
```

Comment:
"preferred" makes it sound a bit optional.  Let's be more explicit: In the case of conflicting requirements, the Cloud Events bindings take precedence over this document.   Or something like that.